src/test/java/org/commcare/backend/session/test/EvalContextTests.java

```diff
@@ -46,6 +46,7 @@ public void testRestrictedEvalContextGeneration1() throws Exception {
         // 1) Confirm that the eval context was restricted properly
         List<String> instancesThatShouldBeIncluded = new ArrayList<>();
         instancesThatShouldBeIncluded.add("casedb");
+        instancesThatShouldBeIncluded.add("commcaresession");
```

Comment:
this doesn't sounds right. `ec` should limit the instances to `instancesNeededByRelevancyCondition`